codes/data/usa/merge_policy_and_cases.py

```diff
@@ -101,16 +101,17 @@ def min_no_nans(x):
 
 def download_and_process_policy_csv():
 
-	policy_data_raw = pd.read_csv(os.path.join(raw_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
+	policy_data_raw = pd.read_csv(os.path.join(int_data_dir,"US_COVID-19_policies.csv"),encoding='latin')
 
 	# A. wrangle polices -> binary variables
 	# get the unchanged vars
 	policy_data = policy_data_raw.loc[:,['date', 'adm0_name', 'adm1_name', 'adm2_name']]
 	
-	policy_data.loc[:,'date'] = policy_data.loc[:,'date'].apply(fix_date)
+	# dates formatted as YYYY-MM-DD so no need to do this anymore
```

Comment:
Don't need to if you'd prefer commented, but should we just delete these lines now that we've got things formatted correctly. Feel free to resolve if you'd rather have them commented

Reply:
i can remove them on the next PR; trying to get it so we can rerun the data rn